linter_exclusions.yml

```diff
@@ -401,6 +401,12 @@ containerapp github-action add:
     service_principal_tenant_id:
       rule_exclusions:
       - option_length_too_long
+containerapp label-history show:
+  rule_exclusions:
+  -  missing_command_test_coverage
+containerapp label-history list:
+  rule_exclusions:
+  -  missing_command_test_coverage
```

Comment:
Why not add some tests for these commands?

Reply:
The commands won't work properly until our next control plane deployment goes out, there was a change in the object structure. I can add tests after that.

Reply:
Got it, thanks~